lib/sailor.js

```diff
@@ -202,15 +201,15 @@ class Sailor {
         self.messagesCount += 1;
 
         const timeStart = Date.now();
-        const traceId = message.properties.headers[AMQP_HEADER_TRACE_ID] || 'unknown';
+        const threadId = message.properties.headers.threadId || 'unknown';
```

Comment:
sorry but I don't understand how trace id is put into next message.
Or this is planned to be separate issue?